cluster/engine_test.go

```diff
@@ -416,8 +416,8 @@ func TestUsedCpus(t *testing.T) {
 
 	for _, hn := range hostNcpu {
 		for _, cn := range containerNcpu {
-			if int(cn) <= hn {
-				mockInfo.NCPU = hn
+			if cn <= hn {
+				mockInfo.NCPU = int(hn)
```

Comment:
`mockInfo.NCPU` is an `int`, and `engine.Cpus` is `int64`, so the conversions would be needed right?